### PR TITLE
Eileen's part4

### PR DESCRIPTION
**Pull request by xa2113**
Finished part 4. Please review :)

### CODE REVIEW

File: src/main/java/com/codingnomads/part_04/Exercise_01.java [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+        int[] intArray = new int[10];
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please print out 10 integers");
+
+        for(int i = 0; i < intArray.length; i++){
+            intArray[i] = scanner.nextInt();
+        }
+
+        System.out.println();
+        System.out.println(intArray[1]);
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_04/Exercise_01.java]: Output here does not match the expectation, missing a "6" :( a loop could help ;)

image

File: src/main/java/com/codingnomads/part_04/Exercise_03.java [hunk 2]
```diff
@@ -16,5 +16,23 @@
 
     public static void main(String[] strings) {
 
+        int[][] numChart = new int[5][10];
+
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_04/Exercise_03.java]: Nice one! 💯 🥇

File: src/main/java/com/codingnomads/part_04/Exercise_06.java [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
  * <p>
  * expected output: 0,2,4,6,8.....100
- */
\ No newline at end of file
+ */
+
+public class Exercise_06{
+    public static void main(String[] args) {
+
+        Queue<Integer> exampleQueue = new LinkedList<>();
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_04/Exercise_06.java]: I believe the expectation here was to use the "PriorityQueue" class but your output does match the requirement so, totally up to you if you want to adjust!

File: src/main/java/com/codingnomads/part_04/Exercise_07.java [hunk 4]
```diff
@@ -1,9 +1,65 @@
...
+
 /**
  The complement of a queue is a stack. It uses first-in, last-out accesssing and is often likened to a stack of plates.
 The first plate put on the table is the last plate used. Create a stack class called Stack that can hold characters. Call
 the method that access the stack push() and pop(). Allow the user to specify the size of the stack when it is created.
 Keep all other members of the Stack class private.
  */
+
+public class Exercise_07{
+    public static void main(String[] args) {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_04/Exercise_07.java]: Oh la la! Works like a charm

File: src/main/java/com/codingnomads/part_04/Exercise_01.java (hunk 1 above)
Comment (xa2113) [reply src/main/java/com/codingnomads/part_04/Exercise_01.java]: Oops! Thanks, nice catch! Fixed this with a loop :)

File: src/main/java/com/codingnomads/part_04/Exercise_06.java (hunk 3 above)
Comment (xa2113) [reply src/main/java/com/codingnomads/part_04/Exercise_06.java]: ahh okay.. I'll just use Queue for now :)

**Pull request merged**